lib/widgets/compose_box.dart

```diff
@@ -1324,13 +1324,14 @@ class _SendButtonState extends State<_SendButton> {
       return;
     }
 
-    final store = PerAccountStoreWidget.of(context);
+    PerAccountStore store = PerAccountStoreWidget.of(context);
```

Comment:
Ah I see, the point is we'll mutate this variable after the async gap, looking up the new value it should have.

nit: Let's move this line to just before the one place this version is used. (And it can go back to `final`.) That way it's clearer that this should only be used before the async gap, and harder to accidentally use it after.